inst/sql/dalys_calculation/raw_remaining_life_expectancy.sql

```diff
@@ -0,0 +1,11 @@
+--- raw life table
+SELECT max(touchstone.version) AS touchstone_version, country, year, age_from, gender, value
+FROM demographic_statistic
+JOIN touchstone_demographic_dataset
+ON touchstone_demographic_dataset.demographic_dataset = demographic_statistic.demographic_dataset
+JOIN touchstone
+ON touchstone.id = touchstone_demographic_dataset.touchstone
+WHERE demographic_statistic.demographic_statistic_type = 11
```

Comment:
join against `demographic_statistic_type` and use the `name` rather than `id`

Reply:
Done.